tests/run_test.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Prophecy\PhpUnit\Tests;
+
+use PHPUnit\TextUI\Command;
+
+require_once dirname(__DIR__) . '/xdebug_filter.php';
```

Comment:
Please move this file in the `tests` dir as it is about the testsuite (this will also automatically exclude it from the archive downloaded by composer as we exclude the test directory)